requirements.txt

```diff
@@ -14,3 +14,4 @@ networkx>=2.2
 pandas
 quandl
 yfinance
+retworkx>=0.4.0
```

Comment:
`retworkx` requirement should be added to `setup.py` as well

Reply:
Done.